File: Hacker Rank/pangrams.py

import math
import os
import random
import re
import sys

alphabet = "abcdefghijklmnopqrstuvwxyz"

string = input()

counter = 0
for char in alphabet:
    if char not in string.lower():
        counter=+1
        break

if(counter==0):
    print("pangram")
else:
    print("not pangram")

File: Hacker Rank/Kotlin/Pangrams.kt
fun main(args: Array<String>){

    var scanner = Scanner(System.`in`)

    var checkString: String = scanner.nextLine()
    var alphabet = "abcdefghijklmnopqrstuvwxyz"

    var length  = alphabet.length
    var count = 0
    for (a in 1..length-1 )
    {

        var result =  checkString.toLowerCase().contains(alphabet[a])
        if(result == true){
            count = count + 1
        }
        else
        {

        }
    }

    if(count == 25)
    {
        println("pangram")
    }
    else
    {
        println("not pangram")
    }
}
